autoload/gfm_syntax/emoji.vim

```diff
@@ -894,7 +894,7 @@ function! gfm_syntax#emoji#complete(findstart, base) abort
     if a:findstart
         return match(getline('.')[0:col('.') - 1], ':[a-z0-9_+-]*$')
     else
-        return filter(map(copy(s:EMOJIS),{_,v->v[0]}), {_,v->match(v, a:base)>=0})
+        return filter(map(copy(s:EMOJIS),{_,v->v[0]}), {_,v->match(v, '[:_]'.a:base[1:])>=0})
```

Comment:
I think calling `filter()` then `map()` is more efficient because filtered list contains fewer elements.